### ISSUE TITLE
NVDA makes Feedback Hub crash when trying to read web content

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#7012]
I was trying to read an article (web content) in the Windows 10 Anniversary Update's Feedback Hub app with NVDA. But each time the focus landed in the web content area, the Feedback Hub would crash. Windows itself and the Feedback Hub app are up-to-date. The bug is reproducible with NVDA 2017.1 and the latest "next" snapshot. I tried with add-ons disabled, but the crash still occurred. If I'm not mistaking, this started occurring only recently - previously I did not have any problems reading articles from Feedback Hub. Also, it seams not to occur with all articles, but with only some of them, like the following: "Maps app gets updated map engine". That one has images in the web content - maybe that is what is causing the crash - I don't know. Also, this does not occur with Narrator, as far as I could test it.

Steps to reproduce it:
1. Make sure that Windows, the Feedback Hub app and NVDA are up-to-date.
2. Launch the Feedback Hub app.
3. Open an article for reading either from the list of recent articles on the app's homepage or from the list of news/announcements in the "Messages" section.
4. Move the focus to the opened article's web content area and start reading it with the arrow keys.

Expected results:
NVDA should be able to read the article with no problems.

Actual results:
The feedback Hub app closes/crashes.

Test environment:
- Operating system: Windows 10 Pro (build 14393.969), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-13946,38d8ee77.
- Feedback Hub version: 1.1702.653.0.
- Processor: Intel Core i5-2320 at 3.00GHz.
- RAM Memory: 8.00GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 3071MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC662 at Intel Cougar Point PCH - High Definition Audio Controller.

**Comment by PratikP1** [comment]
I cannot reproduce this with Windows 10 Pro Insider build 15063. I wonder if Edge is used for rendering the content for announcements etc.

**Comment by gdata1** [comment]
I have now windows 10 15063 such problems do not occur. I think it was a mistake by Microsoft RS2 fixed.

**Comment by jcsteh** [comment]
Similar to (perhaps even the same issue as) #6311. We have two confirmed reports that this is fixed in Windows 10 build 15063, so closing.
